script.js

```diff
@@ -0,0 +1,69 @@
+const addButton = document.getElementById('addButton');
+const booksList = document.getElementById('booksList');
+const addAuthor = document.getElementById('addAuthor');
+const addTitle = document.getElementById('addTitle');
+
+let books;
+
+if (localStorage.getItem('books') === null) {
+  books = [];
+} else {
+  books = JSON.parse(localStorage.getItem('books'));
+}
+
+function displayBooks() {
+  books.forEach((book) => {
+    const li = document.createElement('li');
+    li.innerHTML = `
+    <h2>${book.title}</h2>  
+    <p>${book.author}</p>  
+    <button class="remove">remove</button>
+    <hr>`;
+    booksList.append(li);
+  });
+}
+
+function addBook(e) {
```

Comment:
- [ ] Your `addBook`  function is not working as expected. When the `add` button is clicked a new book is supposed to be added to the collection, with **title** and **author**, however when just the _book title_  or the _ book author_ alone is entered your app accepts the data. Kindly improve upon this aspect by ensuring that both _book title_ and _book author_ are entered before data is accepted.

Reply:
Thanks! Added validation to prevent empty fields.